refinery/ui/source/js/user-file-browser/services/factory.spec.js

```diff
@@ -94,9 +58,10 @@
           {
             organism: {
               facetObj: [
-                { name: 'mouse', count: 4 },
-                { name: 'cat', count: 2 },
-                { name: 'dog', count: 4 }
+                { name: 'mouse', count: 1, assocAttribute: 'organism_Characteristics_generic_s' },
+                { name: 'cat', count: 2, assocAttribute: 'organism_Characteristics_generic_s' },
+                { name: 'mouse', count: 3, assocAttribute: 'organism_Factor_Value_generic_s' },
+                { name: 'dog', count: 4, assocAttribute: 'organism_Factor_Value_generic_s' }
```

Comment:
I'm trying to remember, but are the facet names changing here? Did the `generic` replace just the distinguishing IDs, or `Characteristics` vs `Factor_Value` as well?

Reply:
Never mind: This isn't a change, I don't think.

Reply:
Correct, it's not a change. It's tracking it's internal name vs display name.